### ISSUE TITLE
Multiple rows of channels?

### ISSUE THREAD

**Issue by mxmilkiib** [jack-mixer/jack_mixer#100]
Some day it would be handy to have multiple rows of channels.

**Comment by dsheeler** [comment]
How should this work?  Explicit row creation/deletion?  Dynamic somehow?

**Comment by mxmilkiib** [comment]
Explicit, so the user sets the number of rows, each row takes an equal fraction of the window height, existing channels can then be dragged to a new row. As an extra, maybe a buttom option to hide/collapse each row? The total number of rows can only be reduced if there are no channels still on the next row to be removed.

**Comment by keybreak** [comment]
Yeah explicit is nice idea, in order for one to create something similar to RME TotalMix for instance :smile: 

![](https://archiv.rme-audio.de/images/products/Totalmix_UCX.jpg)

If you have such a device and want to enumerate all of it's ins / outs it's handy to split inputs / virtual outputs visually by raws
